### DISCUSSION TITLE
Can I access super admin dashboard via API?

### DISCUSSION THREAD

**Discussion by happyDuke96** [chatwoot/chatwoot#6337] [Self hosted] [answered]
Hello Chatwoot Team
We use our self hosting, I found out late that I don’t have access to the super admin dashboard through the api. all requests to the api platforms are sent through our backend application, can I somehow get access to the super admin dashboard through the api
![super_admin](https://user-images.githubusercontent.com/105979703/214217887-87295ffa-2810-44bd-bd55-cf06edc5e6d2.PNG)
We need this API to control the user,accounts and other

**Comment by sojan-official** [comment, marked as answer]
We don't have an API for the super admin dashboard. But for account and user management, you can use the platform APIs https://www.chatwoot.com/docs/contributing-guide/chatwoot-apis#platform-apis

**Reply by happyDuke96** [reply]
Thank you very much, if you can an additional question, is there a way to get all the accounts and users and not agents, I want the super admin to have one page and he can see all the accounts and users on this page and if he wants to delete it,in super admin api there is crud api for accounts and users but no api getAll
